redbot/cogs/trivia/trivia.py

```diff
@@ -154,7 +154,7 @@ async def triviaset_allowoverride(self, ctx: commands.Context, enabled: bool):
             )
 
     @triviaset.command(name="usespoilers", usage="<true_or_false>")
-    async def trivaset_use_spoilers(self, ctx: commands.Context, enabled: bool):
```

Comment:
All changes in this file are inaccurate, as the command name is the literal `triviaset`, standing for "trivia setting".

Reply:
I'm sorry, I think my intent was correct but I didn't fix it correctly. The typo was that it's "trivaset" (missing the "i" after the "v"). The problem was that I only fixed the trivia typo and forgot to add "set" with it.

Reply:
Ah, yep I see that now. If you want to make that change but keep `set` in the command name, that would be fine!